deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -717,7 +717,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                     var cell = capacitySetting.currentCell;
                     var scope = this;
                     var provider = capacitySetting.currentProvider;
-                    value = 'cmp_base-ebs';
+                    value = '{{ default_cmp_image }}';
```

Comment:
when instance_type is changed, image name(abstract_name) and image(provider_name or ami) both changes. Can  you make sure to test if image name and image are in sync. For ex: it should not be populating u18 image name and u14 based ami for selection.

Reply:
I reviewed this functionality, it updates to the latest AMI for the correct instance name.  I don't think it's good to be updating this on the user, but it's will keep users up to date, so we can discuss as more of a feature revamp for this form in a follow up.